concordium-contracts-common/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased changes
 
+- Add type for `ContractIndex` and `ContractSubIndex`.
+- Add `micro_ccd` getter for `Amount`.
+- Add `AccountAddress::get_alias` function for finding account aliases.
+- Implement converters to string for `ContractName`, `OwnedContractName` and a `serde` implementation.
+- Implement `Ord` for `OwnedReceiveName`.
+- Change the `serde` implementation for `Address` match the one used by the node.
```

Comment:
Please be more precise. Specify the format.

Reply:
Please see again